docs/manual/en/introduction/How-to-use-node-material.html

```diff
@@ -0,0 +1,815 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<base href="../../../" />
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>How to use Node Material</h1>
+
+		<p>
+		`NodeMaterial API` allows the creation of `Shader Graph` as well as procedural materials compiled for different outputs such as `WebGPU` and `WebGL`.
+		</p>
+
+		<h2>How it works?</h2>
+
+		<p>
+		A `NodeMaterial` consists of `flows` which can have one or more `Nodes`, you can customize a `NodeMaterial` using existing `Nodes` and also create your own.
+		</p>
+
+		<p>
+		All `Nodes` can be connected to each other, and all must have automatic conversion according to the needs of their connections. Each Node and can vary between number of inputs and a single output with dynamic types.
+		`Nodes` can be values such as *Uniforms*, *Constants*, *Attributes of a Geometry* and *Mathematical Functions*.
```

Comment:
I would wrote something like "Nodes can be constants, uniforms, attributes, and functions" (not only mathematical).